src/components/ReportWelcomeText.tsx

```diff
@@ -50,7 +50,7 @@ function ReportWelcomeText({report, policy}: ReportWelcomeTextProps) {
             item !== CONST.IOU.TYPE.INVOICE,
```

Comment:
Not specific to any changed line, but the testing steps 6 and 7 fail because the only welcome text seen on a public room is: `This chat room is for anything **#public-room** related.`

<img width="803" alt="Screenshot 2024-10-08 at 15 25 58" src="https://github.com/user-attachments/assets/8f79984c-7161-45a2-9fe2-d1d4bde7cfd6">

Reply:
We should invite some members to display the `Just use + button to split expense` text.